### PR TITLE
add mermaid documentation

### PR DESCRIPTION
**Pull request by Oz-Okun**
adding mermaid documentation to the docs.

### CODE REVIEW

File: docs/deploying-site/vuepress.md [hunk 1]
```diff
@@ -93,3 +93,94 @@ The default theme also includes a [Badge component](https://v2.vuepress.vuejs.or
...
+    title Timeline
+    2020 : A
+    2021 : B
+         : C
+    2022 : D
+    2023 : E
+```
+
+
+- You can find more options and types of supported charts and diagrams for the Mermaid plugin at the [offical documentation page]("https://mermaid.js.org/intro/").
```
Comment (laurensgroeneveld) [inline docs/deploying-site/vuepress.md:182]: Quotes are not needed here, I think they will be added to the rendered `<a>` element.

```suggestion
- You can find more options and types of supported charts and diagrams for the Mermaid plugin at the [offical documentation page](https://mermaid.js.org/intro/).
```

File: docs/deploying-site/vuepress.md [hunk 2]
```diff
@@ -93,3 +93,94 @@ The default theme also includes a [Badge component](https://v2.vuepress.vuejs.or
...
+
+::: tip
+The mermaid plugin only works on github md files or build!
+:::
```
Comment (laurensgroeneveld) [inline docs/deploying-site/vuepress.md:186]: I find this tip a bit confusing, do you mean that it also works on Github? In that case I would rewrite it to something like this:

```suggestion
::: tip
Mermaid works both in a rendered VuePress site and when viewing the Markdown files on GitHub.
:::
```

**Review by laurensgroeneveld — APPROVED** [review]
Just two suggestions, looks nice otherwise, good to have some examples 👍

File: docs/deploying-site/vuepress.md (hunk 2 above)
Comment (Oz-Okun) [reply docs/deploying-site/vuepress.md:186]: Yes thats what i meant, and your way is better

**Pull request merged**